### PR TITLE
OPSEXP-2712 Add galaxy action for ansible pipelines

### PR DESCRIPTION
**Pull request by pmacius**
### Checklist

- Jira Reference (also in PR title): OPSEXP-2712
- [README](https://github.com/Alfresco/alfresco-build-tools/blob/master/docs/README.md) updated after adding/changing behaviour of an action
- Proposed version increment for [release](https://github.com/Alfresco/alfresco-build-tools/blob/master/docs/README.md#release):
  - [ ] Patch (bugfix)
  - [x] Minor (new feature)
  - [ ] Major (breaking changes)
- External PR link where changes has been tested: https://github.com/Alfresco/devops-engineering-iac/pull/76

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -1523,6 +1524,22 @@ jobs:
...
+
+```yaml
+on:
+  pull_request:
+    branches: [master]
+
+jobs:
+  cleanup:
+    runs-on: ubuntu-latest
+    steps:
```
Comment (gionn) [inline docs/README.md:1539]:
```suggestion
```

just the step is fine if it doesn't requires additional workflow configuration

File: docs/README.md [hunk 2]
```diff
@@ -1523,6 +1524,22 @@ jobs:
...
+      - uses: Alfresco/alfresco-build-tools/.github/actions/galaxy@ref
```
Comment (gionn) [inline docs/README.md:1540]: you can add example usage for the 2 available inputs

File: docs/README.md [hunk 3]
```diff
@@ -1523,6 +1524,22 @@ jobs:
           token: ${{ secrets.GH_TOKEN }}
 ```
 
+### galaxy
```
Comment (gionn) [inline docs/README.md:1527]: move it as per alphabetical sorting (ignore the last one xD)

File: .github/actions/galaxy/action.yml
Comment (gionn) [inline .github/actions/galaxy/action.yml:1]: maybe we can rename it to be more verbose like install-galaxy-deps

**Review by gionn — CHANGES_REQUESTED** [review]

File: .github/actions/galaxy/action.yml
Comment (alxgomz) [reply .github/actions/galaxy/action.yml:1]: install-galaxy-deps made me think it would also install galaxy. What about pull-galaxy-deps?

**Review by gionn — APPROVED** [review]

File: .github/actions/galaxy/action.yml
Comment (gionn) [reply .github/actions/galaxy/action.yml:1]: if it were just `install-galaxy` ok - but `install-galaxy-deps` seems pretty self-explanatory to me

**Pull request merged**